packages/web-ui/src/Book/Thumbnail/SeriesCover/index.tsx

```diff
@@ -0,0 +1,56 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { Palette as VibrantPalette } from 'node-vibrant/lib/color';
+import * as React from 'react';
+import { ThumbnailImageProps } from 'src/Book/ThumbnailImage';
+import { SERIES_BOOK_SCALE } from '../styles';
+import * as styles from './styles';
+// tslint:disable-next-line
+const Vibrant = require('node-vibrant');
```

Comment:
tsconfig에 `"esModuleInterop": true` 추가하시고 그냥 임포트하시면 됩니다. `esModuleInterop`이 `allowSyntheticDefaultImports`도 포함하기 때문에 이 옵션은 지우면 됩니다.

```suggestion
import Vibrant from 'node-vibrant';
```